ipad.css

```diff
@@ -0,0 +1,276 @@
+.body {
+  max-width: 768px;
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  max-width: 768px;
+}
+
+.nav__link {
+  display: none;
+}
+
+.vectors {
+  display: none;
+}
+
+.nav {
+  margin-bottom: 80.32px;
+}
+
+.nav__input {
+  display: block;
+  width: 35px;
+}
+
+.title {
+  max-width: 370px;
+  margin-bottom: 48px;
+}
+
+.title__h1 {
+  margin: 0;
+}
+
+.background {
+  max-width: 768px;
+}
+
+.image__car {
+  width: 87%;
+}
+
+.box {
+  margin: 0 auto 63px;
+  width: 63%;
+  flex-wrap: wrap;
+}
+
+.box__img-1 {
+  width: 65.92px;
+  height: 50.16px;
+}
+
+.box__img-2 {
+  width: 41.56px;
+  height: 64.45px;
+}
+
+.box__item {
+  margin: 0;
+}
+
+.box__item-1 {
+  margin-right: 66px;
+}
+
+.box__text-number {
+  font-size: 50px;
+  line-height: 61px;
+  font-weight: 600;
+}
+
+.mobile-box {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  margin: 0 auto;
+  margin-bottom: 55px;
+}
+
+.mobile-test {
+  margin: 0 auto;
+}
+
+.test {
+  margin: 0;
+}
+
+.test__img-3 {
+  margin: 0;
+  margin-right: 9.6px;
+}
+
+.test__text {
+  font-size: 16px;
+  display: inline-block;
+  margin: 0;
+  margin-right: 34.5px;
+}
+
+.main {
+  max-width: 768px;
+}
+
+.future {
+  display: block;
+}
+
+.future__h1 {
+  height: 500px;
+  max-width: 100%;
+  padding-left: 57%;
+  background-size: 919px;
+  white-space: nowrap;
+}
+
+.section {
+  padding-top: 55px;
+}
+
+.section__future {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  width: 100%;
+}
+
+.mobile__section {
+  max-width: 300px;
+  margin:  0 116px 60px 34px;
+}
+
+ .mobile__section_icon {
```

Comment:
Codacy found an issue: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla-n/pullRequest?prid=5749910)